ui/app/mainui/AppMain.qml

```diff
@@ -1967,7 +1967,7 @@ Item {
                 if (showQR.showSingleAccount || showQR.showForSavedAddress) {
                     return showQR.selectedAccount
                 }
-                return selectedReceiverAccount.item ?? ModelUtils.get(appMain.transactionStore.accounts, 0)
+                return selectedReceiverAccount.item ?? SQUtils.ModelUtils.get(appMain.transactionStore.accounts, 0)
```

Comment:
OK, much better... but... how does a fix inside `ReceiveModal` have an impact on that user icon? CC @alexjba

Reply:
Here the root cause of the issue is in "Utils.qml" two files Utils.qml ui\imports\utils\Utils.qml and ui\StatusQ\src\StatusQ\Core\Utils\Utils.qml and recent changes introduced StatusQ's Utils.qml and it resulted in qtquick engine 'overwriting' previous and correct Utils.qml with one from StatusQ. Adding import qualifier resolves the problem

Reply:
Alright... but it still doesn't explain my question :) Anyways, can you pls look into other occurrences of the two `Utils` being imported unaliased throughout the code? I'm pretty sure we have more cases like this